src/ray/object_manager/object_manager.cc

```diff
@@ -109,6 +109,8 @@ ray::Status ObjectManager::SubscribeObjDeleted(
 }
 
 ray::Status ObjectManager::Pull(const ObjectID &object_id) {
+  RAY_LOG(DEBUG) << "[ObjectManager] Pull on " << client_id_ << " of object "
+                 << object_id;
```

Comment:
the `[ObjectManager]` may be redundant, because glog should already print the file name, which is `object_manager.cc`

Reply:
Thanks, I'll remove these.